config/environment_provisioning/environment_provisioning.yaml

```diff
@@ -0,0 +1,44 @@
+---
+kind: CronJob
+apiVersion: batch/v1
+metadata:
+  name: spacerequest-garbage-collector
+spec:
+  schedule: "0 5 * * *" # every day at 5AM UTC
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          serviceAccountName: appstudio-pipeline
+          containers:
+            - name: cleanup-container
+              image: quay.io/redhat-user-workloads/rhtap-o11y-tenant/tools/tools:3e4c9a30e734aa34f57a084633ccfbb73d466fbd
```

Comment:
shouldn't be this an released image, not image taken directly from user-workloads?

Reply:
I don't think we need this particular image. We can do with something more general-purpose with `oc` installed.